java/client/build.gradle

```diff
@@ -181,19 +181,19 @@ publishing {
     publications {
         mavenJava(MavenPublication) {
             from components.java
-            groupId = 'software.amazon.glide'
-            artifactId = 'glide-for-redis'
+            groupId = 'io.valkey'
+            artifactId = 'valkey-glide'
```

Comment:
This will end up something like: 
https://repo.maven.apache.org/maven2/io/valkey/valkey-glide/maven-metadata.xml